modules/mobile/docs/schemas/claimEventTimeline.yml

```diff
@@ -57,4 +57,9 @@ properties:
     example: null
   filename:
     type: string
-    example: '7B434B58-477C-4379-816F-05E6D3A10487.pdf'
\ No newline at end of file
+    example: '7B434B58-477C-4379-816F-05E6D3A10487.pdf'
+  document_id:
+    type: string
+    description: 'Only used for other_documents_list type event for untracked documents'
```

Comment:
I think we want this doc id included for all docs not just untracked ones

Reply:
tracked documents are already being tracked within the `modules/mobile/docs/schemas/claimDocument.yml`. Tracked items has a `documents` array that i've added document_id to. Untracked items are generated as a generic event so I had to add the document_id to the `eventTimeline` so it'll be null most of the time when not an untracked document. its not pretty but this is the format FE expects